plugins/woocommerce-admin/client/products/categories/product-details-category.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { TextControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ProductCategoryLayout } from '../layout/product-category-layout';
+
+export const ProductDetailsCategory: React.FC = () => {
+	return (
+		<ProductCategoryLayout
```

Comment:
Love this direction!

One nitpick I have is that `ProductCategory` made me think this was a "product category" (the taxonomy) at first.  Do these correlate with "sections" or some other semantic term to identify what this is?

Reply:
Good point, I like `Sections` that might be the better (less confusing) name to use.

Reply:
Addressed here: https://github.com/woocommerce/woocommerce/pull/34113/commits/a036816809d8e113282de8a20d79993f8d049e46